src/ol/Geolocation.js

```diff
@@ -174,9 +174,12 @@ class Geolocation extends BaseObject {
    * @param {PositionError} error error object.
    */
   positionError_(error) {
-    error.type = EventType.ERROR;
```

Comment:
We this change, we lost the detail of the error (error code, message) as defined in https://developer.mozilla.org/en-US/docs/Web/API/PositionError

Reply:
First the `PositionError` type does not contain a `type` property.
Then, you were returning the error object as the event, that was an error IMO.
Nothing is done with the error object in the end.

Where would you like to store `error.code` ?